lecture02-conda-python/slides.tex

```diff
@@ -227,64 +228,70 @@ \section{Jupyter}
 
 \section{Homework assignment}
 
-\begin{frame}{Assignment (I)}
+\begin{frame}[fragile]{Homework assignment (I)}
 
 	\begin{itemize}
-		\item If you haven't done so yet, fork the homework repository at https://github.com/inwe-boku/homework-scientific-computing
-		\item You may have to fetch the latest changes to your repo
-		\item Install conda and jupyter lab on your computer, run jupyter lab in the directory of the forked repository
-		\item Use Python to create an encrypted file with the full name of the team member, the registration number (Matrikelnummer), and the git username. For that purpose, use the public key \textit{public\_key.pem} stored in the homework repository in \textit{homework02-conda-python}
-		\item The filename of the encrypted file should be \textit{group-<name of group>} \textit{<name of group>} is your self-chosen group name.
+		\item During your last homework assignment, one of your team members forked the homework repository:\\
+            https://github.com/inwe-boku/homework-scientific-computing
+		\item Fetch the latest changes from the upstream:
+            {\scriptsize
+            \begin{verbatim}
+cd path/to/homework-scientific-computing
+git remote add upstream https://github.com/inwe-boku/homework-scientific-computing/
+git pull --no-edit upstream master\end{verbatim}
+            }
+		\item Install conda and Jupyter and the package cryptography on your computer, run Jupyter in the directory of the
```

Comment:
if students install python with anaconda, the cryptography package is automatically installed. So if they just copy the conda install command from the slides, there is no need to install cryptography.